src/screens/WasteCollectionScreen.js

```diff
@@ -0,0 +1,299 @@
+    }
+  );
+
+  const renderSuggestion = useCallback(
+    ({ item }) => (
+      <TouchableOpacity
+        onPress={() => {
+          setInputValue(getStreetString(item));
+          Keyboard.dismiss();
+        }}
+      >
+        <RegularText>{getStreetString(item)}</RegularText>
+      </TouchableOpacity>
+    ),
+    [setInputValue]
+  );
+
+  // TODO: disabled until backend is ready
+  // const goToReminder = useCallback(
+  //   () => navigation.navigate('WasteReminder', { wasteTypes: usedTypes, streetData }),
+  //   [navigation, usedTypes, streetData]
+  // );
+
+  useEffect(() => {
+    if (!addressesData) {
+      return;
+    }
+
+    const item = addressesData.find(
+      (street) => getStreetString(street).toLowerCase() === inputValue.toLowerCase()
+    );
+
+    setSelectedStreetId(item?.id);
+  }, [inputValue, setSelectedStreetId, addressesData]);
+
+  if (loading || typesLoading) {
+    return (
+      <LoadingContainer>
+        <ActivityIndicator color={colors.accent} />
+      </LoadingContainer>
+    );
+  }
+
+  const filteredStreets = filterStreets(inputValue, addressesData);
+
+  return (
+    <SafeAreaViewFlex>
+      <ScrollView keyboardShouldPersistTaps="always">
+        <WrapperWithOrientation>
+          <Autocomplete
+            containerStyle={styles.autoCompleteContainer}
+            data={filteredStreets}
+            defaultValue={inputValue}
+            onChangeText={(text) => setInputValue(text)}
+            placeholder="Straße"
+            renderItem={renderSuggestion}
+          />
+          <View style={styles.topMarginContainer}>
+            <Calendar
+              markedDates={getMarkedDates(parsedTypesData, streetData)}
+              markingType="multi-dot"
+              renderArrow={renderArrow}
+              theme={{ todayTextColor: colors.primary }}
+            />
+            <WasteCalendarLegend data={usedTypes} />
+          </View>
+          {!selectedStreetId && (
+            <Wrapper>
+              <RegularText center>{texts.wasteCalendar.hint}</RegularText>
+            </Wrapper>
+          )}
+          {/*
+          TODO: disabled until backend is ready
+          {!!data && selectedStreetId && (
+            <Wrapper>
+              <Button title={texts.wasteCalendar.configureReminder} onPress={goToReminder} />
+            </Wrapper>
+          )}
+          */}
+        </WrapperWithOrientation>
+      </ScrollView>
+    </SafeAreaViewFlex>
+  );
+};
+
+const styles = StyleSheet.create({
+  autoCompleteContainer:
+    device.platform === 'android'
+      ? {
+          alignSelf: 'center',
+          flex: 1,
+          position: 'absolute',
+          top: 0,
+          width: '100%',
+          zIndex: 1
+        }
+      : {},
+  icon: {
+    paddingHorizontal: normalize(14)
+  },
+  topMarginContainer:
+    device.platform === 'android'
+      ? {
+          marginTop: 40
```

Comment:
works fine across different android devices without `normalize`?

Reply:
it did without the styling of the text input. now it does not anymore so i added it and increased it slightly as well.